Index: github/config/mutt/extracturl/extract_url.pl
===================================================================
--- github.orig/config/mutt/extracturl/extract_url.pl
+++ github/config/mutt/extracturl/extract_url.pl
@@ -278,6 +278,7 @@ sub renderuri {
 }
 sub sanitizeuri {
 	my($uri) = @_;
+	return $uri;
 	if ($sanitize_reserved) {
 		$uri =~ s/([^a-zA-Z0-9_.~%:\/-])/sprintf("%%%X",ord($1))/egs;
 	} else {
